src/manifest.js

```diff
@@ -34,7 +34,7 @@ export const parseManifest = ({
   manifestString,
   customTagParsers = [],
   customTagMappers = [],
-  experimentalLLHLS
+  llhls = true
```

Comment:
I think this default should be at a higher level (probably in src/videojs-http-streaming.js), as, as it stands, it looks like playlist-loader will set it to false if left unspecified.